models/Post.php

```diff
@@ -112,12 +113,16 @@ class Post extends Model
      */
     public function filterFields($fields, $context = null)
     {
+        $user = BackendAuth::getUser();
+        
+        if ($context == 'create') {
```

Comment:
@rubenvanerk Please make this an explicit check via `===`
```suggestion
        if ($context === 'create') {
```